models/insight.go

```diff
@@ -0,0 +1,6 @@
+package models
+
+type Insight struct {
```

Comment:
After connecting to your database:
```
baprmzsc=> SELECT * FROM assets;
 id | user_id | favourite 
----+---------+-----------
(0 rows)

baprmzsc=> SELECT * FROM insights;
 id | user_id | favourite |                                text                                 
----+---------+-----------+---------------------------------------------------------------------
  4 |       1 | t         | 40% of millenials spend more than 3hours on social media daily
  5 |       1 | t         | 50% of females in age 30-35 spend more than 8hours on running weekl
  6 |       1 | f         | 60% of females in age 50-60 spend more than 3 on internet daily
(3 rows)
```
Can you please elaborate on how table `assets` is utilized by your application?